05-cluster-prerequisites.md

```diff
@@ -35,14 +35,25 @@ Following the steps below will result in the provisioning of the shared Azure re
+  >
+  > - Azure Container Registry
+  > - Azure Application Gateway
+  > - Azure Key Vault
+  >
+  >  In the future, the app team is considering to enforce the Azure resources to send their Diagnostics logs with an Azure Policy as well as granting different users access rights to keep the data in isolation using Azure RBAC, something that is possible within a single workspace.
+  >
+  >  :bulb:  Azure Log Analytics can be modeled in different ways depending on your organizational needs. It can be _centralized_ as in this reference implementation, or _decentralized_ or a combination of both, which is known as _hybrid_. Azure Log Analytics workspaces are in a geographic location for data storage; consider, for high availability, a distributed solution as the recommended approach instead. If you opt for a _centralized_ solution, you need to be sure that the geo data residency is not going to be an issue, and be aware that cross-region data transfer costs will apply.
+  >
+  > :book: The app team is starting to lay down the groundwork for a high availability, and they know that centralizing Azure resources might introduce single point of failures. Therefere, the app team is tasked to assess how the resources could be shared efficiently without losing reliablity. This is turn for the Azure Container Registry with the additional problematic that container images can be heavy artifacts to deal with, so they are considering having presence in multiple regions as well. This requirement comes in hand with distributing the exact same container images as this is for tween clusters that happens to live in different regions. Although managing registry instances per region individually will mitigate the risk of having a region down, this approach won’t be falling back automatically nor replicating images by requiring in both cases manual intervention and/or additional procedures.  That is the reason why, the team is now up for the `Premium` tier that offers [Geo-Replication](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-geo-replication) as a built-in feature giving them the ability to share a single registry, more availability, and at same time reduce the network latency.  The app team plan to geo replicate the registries to the same regions where their AKS clusters are going to be deployed (`East US 2` and `Central US`), as this is the recommendation to optimize the DNS resolution. They pay close attention to the replicated regions, as they want no more regions than needed since the business unit (bu0001) incurs `Premium` SKU registry fees for each region they desire to geo-replicate. Under this configuration, they will pay *two* times `Premium` per month to get region proximity to ensure no extra network egress fees from distant regions.
```

Comment:
> This requirement comes in hand with distributing the exact same container images as this is for tween clusters that happens to live..

> This is turn for the Azure Container Registry with the additional problematic that container images can be heavy artifacts to deal with, so they a

Something seems a bit off in these two sentences to me. I think it's just wording, not message.




```suggestion
  > :book: The app team is starting to lay down the groundwork for a high availability, and they know that centralizing Azure resources might introduce single point of failures. Therefore, the app team is tasked to assess how the resources could be shared efficiently without losing reliability. This is turn for the Azure Container Registry with the additional problematic that container images can be heavy artifacts to deal with, so they are considering having presence in multiple regions as well. This requirement comes in hand with distributing the exact same container images as this is for tween clusters that happens to live in different regions. Although managing registry instances per region individually will mitigate the risk of having a region down, this approach won’t be falling back automatically nor replicating images by requiring in both cases manual intervention and/or additional procedures.  That is the reason why, the team selected the **Premium** tier that offers [Geo-Replication](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-geo-replication) as a built-in feature; giving them the ability to share a single registry, higher availability, and at same time reducing network latency. The app team plans to geo-replicate the registries to the same regions where their AKS clusters are going to be deployed (`East US 2` and `Central US`), as this is the recommendation to optimize the DNS resolution. They pay close attention to the replicated regions, as they want no more regions than needed since the business unit (bu0001) incurs Premium SKU registry fees for _each region_ they geo-replicate to. Under this configuration, they will pay *two* times Premium per month to get region proximity and to ensure no extra network egress fees from distant regions.
```